content/blog/entries/gsoc2019-arimadian/contents.lr

```diff
@@ -0,0 +1,28 @@
+title: Ari Madian - My GSoC 2019 Blog
```

Comment:
The title should be specific to this blog post, I'd recommend including your project and the week number in this title.